app/controllers/report_controller.rb

```diff
@@ -10,37 +10,22 @@ def rsolr_request_error(exception)
   end
 
   def data
-    params[:sord] ||= "asc"
-    rows_per_page = params[:rows] ? params.delete(:rows).to_i : 10
-    params[:per_page] = rows_per_page * [params.delete(:npage).to_i, 1].max
+    params[:per_page] = params[:per_page] ? params[:per_page].to_i : Report::ROWS_PER_PAGE
+    params[:page] ||= 1
```

Comment:
The client-side tooling will automagically increment this param as it progressively loads report results via the server-side API (`ReportController` & `Report` model).

Reply:
this would make a good comment in the code.

Reply:
Added.